packages/static-build/test/fixtures/02-cowsay-sh/node22sh/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "engines": {
-    "node": "16.x"
+    "node": "22.x"
```

Comment:
The specific versions don't matter in this fixture, just that each version is different and both are available on the same image. Soon only node 18 will be available on the non-AL2 container, but 20/22 are both in the AL2 containers.